packages/ui/src/working-groups/hooks/useWorkerEarnings.ts

```diff
@@ -14,6 +14,7 @@ export const useWorkerEarnings = (workerId: string) => {
 
   const currentDayEarnings = data?.rewardPaidEvents
     .filter((event) => {
+      // TODO use rewardPerBlock
```

Comment:
not 100% comfortable now leaving a TODO here

Reply:
I think it's fine since the issue which should address this TODO is still open: #3181